src/gui/symbology/qgsgraduatedsymbolrendererwidget.cpp

```diff
@@ -777,15 +782,14 @@ void QgsGraduatedSymbolRendererWidget::updateSymbolsFromWidget()
         if ( selectedIndexes.count() > 1 )
         {
           //if updating multiple ranges, retain the existing range colors
-          newRangeSymbol->setColor( mRenderer->ranges().at( rangeIdx ).symbol()->color() );
+          newRangeSymbol->setColor( mRenderer->ranges()[ rangeIdx ].symbol()->color() );
```

Comment:
Why did you change this? If I read the docs right, the [] operator returns a non const reference while at() does. You are not modifying there reference here so at() looks more appropriate to me.

Reply:
I will revert the change